pydsa/tests/test_bsf.py

```diff
@@ -0,0 +1,6 @@
+from pydsa import bsf
+
+def test_bsf():
+	graph = [[1,2],[2],[0,3],[3]]
```

Comment:
Usually a dictionary with nodes as `keys` and `value` as the nodes connected to that node is a better representation.
For Example:

```
graph = {'A': set(['B', 'C']),
         'B': set(['A', 'D', 'E']),
         'C': set(['A', 'F']),
         'D': set(['B']),
         'E': set(['B', 'F']),
         'F': set(['C', 'E'])}
```